lib/util.py

```diff
@@ -77,24 +77,30 @@ def cache_cookies(cache):
         cache.set(CACHE_COOKIES, _cookie_jar.dump())
 
 
-def request(url, headers={}):
+def request(url, headers={}, errors_are_useful=False):
```

Comment:
Do we really need this argument ? I'd read the error every time it occurs, would solve some crashes in other resolvers and plugins.